go/kbfs/libkbfs/prefetcher_test.go

```diff
@@ -1847,6 +1857,8 @@ func TestPrefetcherReschedules(t *testing.T) {
 		context.Background(), individualTestTimeout)
 	defer cancel()
 	kmd := makeKMD()
+	prefetchDoneCh := make(chan struct{})
```

Comment:
It's a bit confusing that this is declared all the way up here, when it isn't used until the `TogglePrefetcher` much lower. Or did you intend to use it in the `TogglePrefetcher` on line 1864?

Reply:
@jzila it's actually used in the `defer` right below this line, which is the important thing.  Because it needs to be closed _after_ the deferred `shutdownPrefetcherTest` call a few lines below that.  If it's shutdown before, there's a potential for a data race (see the linked HOTPOT issue) where the prefetcher tries to write to that channel after it's been closed by the test.  I'll add a comment there explaining it.

Reply:
Got it, thanks.